src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/MassTransit/NewRelicFilter.cs

```diff
@@ -54,7 +54,7 @@ public async Task Send(ConsumeContext context, IPipe<ConsumeContext> next)
 
             var segment = transaction.StartMessageBrokerSegment(mc, MessageBrokerDestinationType.Queue, MessageBrokerAction.Consume, MessageBrokerVendorName, queueData.QueueName);
 
-            await next.Send(context);
+            await next.Send(context).ConfigureAwait(false);
```

Comment:
I don't know if this would cause problems for other filters in the pipeline if there is a synchronization context.